newIDE/app/src/EventsFunctionsExtensionsLoader/MetadataDeclarationHelpers.js

```diff
@@ -894,9 +900,13 @@ const declarePropertyInstructionAndExpression = (
       entityMetadata.addScopedAction(
         actionName,
         propertyLabel,
-        i18n._(t`Update the property value for ${uncapitalizedLabel}`),
         i18n._(
-          t`Set property value for ${uncapitalizedLabel} of _PARAM0_ to _PARAM${valueParameterIndex}_`
+          t`Update the property value for ${'"' + uncapitalizedLabel + '"'}.`
```

Comment:
Should the quotes be inside the translation? Not all languages use them.

Reply:
When I tried the translation wasn't working. I don't understand why.

Reply:
Oh that's very strange then

Reply:
Actually, it works. I must have tried to concatenate translatable objects at some point.